addons/stock_transport/models/stock_picking_batch.py

```diff
@@ -0,0 +1,112 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class StockPickingBatch(models.Model):
+    _inherit = 'stock.picking.batch'
+
+    @api.model
+    def _default_end_date(self):
+        self.end_date = fields.datetime.now()
+
+    vehicle_id = fields.Many2one('fleet.vehicle', string="Vehicle")
+    vehicle_category_id = fields.Many2one(
+        'fleet.vehicle.model.category', string="Vehicle Category", compute='_compute_vehicle_category_id', inverse='_inverse_vehicle_category_id', store=True)
+    dock_id = fields.Many2one('stock.location', string="Dock Location", domain="[('is_a_dock', '=', True)]")
+    vehicle_details = fields.Char(
+        "Vehicle Details", compute='_compute_vehicle_details')
+    max_weight = fields.Float(string="Max Weight (Kg)",
+                              related='vehicle_category_id.max_weight')
+    max_volume = fields.Float(string="Max Volume (m³)",
+                              related='vehicle_category_id.max_volume')
+    driver_id = fields.Many2one(
+        related="vehicle_id.driver_id", string="Driver")
+    used_weight_percentage = fields.Float(
+        string="Weight %", compute='_compute_used_weight_percentage', store=True)
+    used_volume_percentage = fields.Float(
+        string="Volume %", compute='_compute_used_volume_percentage', store=True)
```

Comment:
No need to store it

Reply:
since I have computed these fields and these were used in pivot view so made it store=True.